api/src/main/java/org/openmrs/api/db/ConceptDAO.java

```diff
@@ -393,6 +393,12 @@ public Integer getCountOfConcepts(String phrase, List<Locale> locales, boolean i
 	public ConceptSource getConceptSourceByUniqueId(String uniqueId);
 
 	/**
+	 * @see org.openmrs.api.ConceptService#getConceptSourceByHL7Code(java.lang.String)
+	 */
+	
```

Comment:
Actually there was a linebreak used after every test in this file. So I went along with it